make_windows_package.sh

```diff
@@ -47,6 +47,7 @@ cp -R octsympy ${WINDIRTMP}
   ./bootstrap
   ./configure
   make )
+cp bin/winwrapy.bat ${WINDIRTMP}/bin/
```

Comment:
Ok, I see.

I'm still dropping that line.  These scripts probably need some work: they are intended to run from *another directory*, checkout a clean copy and package it.  I agree that's not ideal for testing.  Guess they could be improved as part of #254.  Probably incorporated into the makefile itself.

Although, if memory serves the @oheim's Maintainer's Makefile still needs changed committed (although not pushed to remote repo).

Thanks again for this.